app/controllers/organisations_controller.rb

```diff
@@ -46,13 +46,17 @@ def destroy
 
 private
 
+  def organisation_software_path
+    organisation_software_instances_path(organisation)
+  end
+
   # Use callbacks to share common setup or constraints between actions.
   def organisation
-    @organisation ||= authorize Organisation.find(params[:id])
+    @organisation ||= authorize Organisation.find_by(tag: params[:id])
```

Comment:
Not related to this change, but I just happened to be reading something about using the duck operator for initialisation vs using the `initialize` method earlier today, wondered what your thoughts are? https://dev.to/matmooredev/please-stop-using-all-the-time-2l8b

Reply:
fair enough, thanks for explaining!